material/static/material/js/materialize.js

```diff
@@ -2033,8 +2033,11 @@ $(document).ready(function(){
         }
 
         // Add Touch Area
-        var dragTarget = $('<div class="drag-target"></div>');
-        $('body').append(dragTarget);
+        var dragTarget = $('.drag-target');
```

Comment:
this won't work, since materialize.js is updated by gulp from materializecss project